airbyte-integrations/connectors/source-fullstory/metadata.yaml

```diff
@@ -14,5 +14,8 @@ data:
     oss:
       enabled: true
   releaseStage: alpha
-  supportUrl: https://docs.airbyte.com/integrations/sources/fullstory
+  documentationUrl: https://docs.airbyte.com/integrations/sources/fullstory
+  tags:
+    - language:low-code
```

Comment:
Is low-code a language?  Maybe "low-code:true"?

Reply:
I think lowcode counts as a language tag for now.

If we start programming on these tags then we can haul it into a standalone field